src/tzdata/compile.jl

```diff
@@ -69,55 +69,120 @@ const DAYS = Dict(
     "Mon" => 1, "Tue" => 2, "Wed" => 3, "Thu" => 4, "Fri" => 5, "Sat" => 6, "Sun" => 7,
 )
 
+const LAST_DAY_OF_WEEK = Dict{String, Function}()
+
 # Create adjuster functions such as "lastSun".
 for (abbr, dayofweek) in DAYS
-    sym = Symbol("last" * abbr)
-    @eval (
+    str = "last" * abbr
+    sym = Symbol(str)
+    LAST_DAY_OF_WEEK[str] = @eval begin
         function $sym(dt)
             return dayofweek(dt) == $dayofweek &&
             dayofweekofmonth(dt) == daysofweekinmonth(dt)
         end
-    )
+    end
+end
+
+# Generate various DateFormats based the number of periods provided.
+const UNTIL_FORMATS = let
+    parts = split("yyyy uuu dd HH:MM:SS", ' ')
+    date_formats = Vector{DateFormat}(undef, length(parts))
+    for i in 1:length(date_formats)
+        date_formats[i] = DateFormat(join(parts[1:i], ' '))
+    end
+    date_formats
 end
 
+
+
 isflag(flag::Char) = flag in ('w', 'u', 's')
 
+
+"""
+    tryparse_dayofmonth(str::AbstractString) -> Union{Function,Nothing}
+
+Parse the various day-of-month formats used within tzdata source files.
+
+```julia
```

Comment:
This should be `julia-repl`, which provides better highlighting for simulated REPL sessions.

Reply:
TIL

Reply:
Oddly enough, I learned about `julia-repl` while fixing up docs for this package!

Reply:
I worked with Morten to get it added to the JavaScript library that provides Documenter's highlighting, which is why I know about it. :slightly_smiling_face: